plsync/sites.py

```diff
@@ -115,6 +115,8 @@
     makesite('lga06','128.177.119.192','2001:438:fffd:2b::',   'New York_NY', 'US', 40.766700, -73.866700, user_list, count=4, arch='x86_64', nodegroup='MeasurementLabCentos'),
     makesite('lga07','66.151.223.128', '2600:c0f:2002::',      'New York_NY', 'US', 40.766700, -73.866700, user_list, nodegroup='MeasurementLabCentos'),
     makesite('lhr01','217.163.1.64',   '2001:4c08:2003:3::',   'London', 'GB', 51.469700, -0.451389, user_list, nodegroup='MeasurementLabCentos'),
+    makesite('lhr02','80.239.170.192',   '2001:2030:33::',   'London', 'GB', 51.469700, -0.451389, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
+    makesite('lhr03','77.67.114.196',   '2001:0668:001F:0061::',   'London', 'GB', 51.469700, -0.451389, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
```

Comment:
Something doesn't look right about this network address. The last octet of a /26 prefix can only be one of [0, 64, 128, 192]. I'm assuming that the 196 was a fat finger and that it should be 192. However, you'll want to verify this 100% with Mike.